presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -67,6 +67,24 @@ The configuration files must exist on all Presto nodes. If you are
 referencing existing Hadoop config files, make sure to copy them to
 any Presto nodes that are not running Hadoop.
 
+Accessing Hadoop clusters protected with Kerberos authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Kerberos authentication is currently supported for both HDFS and Hive metastore.
+
+However there are still few limitations:
+
+* Kerberos authentication is supported only for ``hive-hadoop2`` and `hive-cdh5` connectors.
+* Kerberos authentication by ticket cache is not yet supported.
+
+Please refer to  `Configuration Properties`_ section for configuration details.
+
+.. note::
+
+    If your ``krb5.conf`` location is different than ``/etc/krb5.conf`` you must set it
```

Comment:
This is not ideal because we lose the ability to validate options that are set in this manner. I guess it's fine for now, but we should figure out a way to make the option required by the hive connector and the one required by Presto coexist.